packages/node_modules/@webex/plugin-meetings/src/meetings/request.js

```diff
@@ -26,6 +26,14 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       });
   }
 
+  /**
+     *  fetch geoHit for the user
+     * @returns {Object} geoHintInfo
```

Comment:
Should this be `@returns {Promise<object>} geoHintInfo`?

Reply:
done, changed the logic to add country code only when its available